moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -709,12 +786,78 @@ void MotionPlanningFrame::createSceneInteractiveMarker()
         robot_interaction::make6DOFMarker(std::string("marker_") + sel[0]->text().toStdString(), shape_pose, 1.0);
     int_marker.header.frame_id = planning_display_->getRobotModel()->getModelFrame();
     int_marker.description = sel[0]->text().toStdString();
-
+    
     rviz::InteractiveMarker* imarker = new rviz::InteractiveMarker(planning_display_->getSceneNode(), context_);
     interactive_markers::autoComplete(int_marker);
+
+    // Resize the marker scale
```

Comment:
the code below looks the same as above. maybe refactor to avoid the duplicate code?

Reply:
I agree with this, and as @rhaschke mentioned, I will replace this part to the `computeShapeBoundingSphere()` function.

Reply:
Additionally to using `computeShapeBoundingSphere()`, you should factor the marker adjustment code into its own function.

Reply:
@rhaschke 
The "own function" you mean, I should create original function in this class to utilize `computeShapeBoundingSphere()` function and to adjust the marker. Is that right ?

Reply:
Exactly.

Reply:
I created the `resizeInteractiveMarker()` function, and to utilize it.